src/bindings/html/dom.js

```diff
@@ -42,15 +42,14 @@ export function getAttributes(element) {
 }
 
 function getTranslatables(element) {
-  const nodes = [];
+  const nodes = [...element.querySelectorAll('*[data-l10n-id]')];
```

Comment:
I really like this way of transforming a `NodeList` into an array! It does seem a bit slower than the traditional methods (cf. http://jsperf.com/qsa-slice-vs-spread) but I'm all for keeping it here.